lib/logger.js

```diff
@@ -121,6 +121,29 @@ var streamConverter = {
 };
 var streamConverterList = Object.keys(streamConverter);
 
+Logger.prototype._addErrorHandler = function() {
+    var self = this;
+    self._logger.on('error', function(err, failedStream) {
+        // If some fatal error occurred in one of the logger streams,
+        // we can't do much, so ignore.
+
+        if (err.code === 'ENOSPC') {
```

Comment:
We normally still log to logstash, so imho we should err on the side of caution for local logging. There is no way for us to distinguish temporary failures from permanent ones, so I think we should just stop logging on any error. In any case, write failures are an exceptional situation, so this shouldn't affect normal operation.